packages/core/src/index.ts

```diff
@@ -4,13 +4,14 @@ import { Effection } from './effection';
 
 export { State, StateTransition } from './state-machine';
 export { createTask, Task, TaskOptions, TaskInfo, TaskTree } from './task';
-export { Operation, Resource } from './operation';
+export { Operation, Resource, OperationObject } from './operation';
```

Comment:
@cowboyd Why we don't export other operation types? Are there any rules what stuff should be exported from lib and what shouldn't?

Reply:
I think we want to discourage using any of the sub-operation types (with the possible exception of Resource). However, in the same way that `Iterable` and `Iterator` are different, we may want to export the `OperationObject` interface as `Operator`. For the time being, let's _not_ export it.